pkg/services/grafana-apiserver/service.go

```diff
@@ -163,7 +162,19 @@ func (s *service) start(ctx context.Context) error {
 		logger.Error(err, "failed to set log level")
 	}
 
-	o := options.NewRecommendedOptions("", unstructured.UnstructuredJSONScheme)
+	// Get the list of groups the server will support
+	builders := s.builders
+
+	groupVersions := make([]schema.GroupVersion, 0, len(builders))
+	// Install schemas
+	for _, b := range builders {
+		groupVersions = append(groupVersions, b.GetGroupVersion())
+		if err := b.InstallSchema(Scheme); err != nil {
+			return err
+		}
+	}
+
+	o := options.NewRecommendedOptions("/registry/grafana.app", Codecs.LegacyCodec(groupVersions...))
```

Comment:
is "/registry/grafana.app" just a unique string?

Reply:
yeah. it's the etcd prefix